src/aead/aes_gcm.rs

```diff
@@ -58,11 +58,9 @@ pub static AES_256_GCM: aead::Algorithm = aead::Algorithm {
     open: aes_gcm_open,
 };
 
-fn aes_gcm_init(ctx_buf: &mut [u8], key: &[u8])
-                -> Result<(), error::Unspecified> {
+fn aes_gcm_init(ctx_buf: &mut [u8], key: &[u8]) -> Result<(), error::Unspecified> {
     bssl::map_result(unsafe {
-        evp_aead_aes_gcm_init(ctx_buf.as_mut_ptr(),
-                              ctx_buf.len(), key.as_ptr(), key.len())
+        evp_aead_aes_gcm_init(ctx_buf.as_mut_ptr(), ctx_buf.len(), key.as_ptr(), key.len())
```

Comment:
Is this just the result of running rustfmt? Let's keep the changes in this PR specifically to the tests, so let's remove this.